third_party/move/move-compiler-v2/tests/bytecode-generator/wildcard3.exp

```diff
@@ -26,31 +26,35 @@ module 0xc0ffee::m {
 public fun m::bar() {
      var $t0: m::S
      var $t1: bool
-     var $t2: m::S
   0: $t1 := false
   1: $t0 := pack m::S($t1)
-  2: $t2 := infer($t0)
-  3: return ()
+  2: return ()
 }
 
 
 [variant baseline]
 public fun m::foo($t0: m::S) {
-     var $t1: m::S
-  0: $t1 := infer($t0)
-  1: return ()
+  0: return ()
 }
 
 
 Diagnostics:
-error: value of type `m::S` does not have the `drop` ability
-  ┌─ tests/bytecode-generator/wildcard3.move:5:13
+warning: Unused assignment to `s`. Consider removing or prefixing with an underscore: `_s`
+  ┌─ tests/bytecode-generator/wildcard3.move:9:17
+  │
+9 │         let s = S{};
+  │                 ^^^
+
+
+Diagnostics:
+error: local `s` of type `m::S` does not have the `drop` ability
+  ┌─ tests/bytecode-generator/wildcard3.move:5:18
   │
 5 │         let _ = s;
-  │             ^ implicitly dropped here since it is no longer used
+  │                  ^ implicitly dropped here since it is no longer used
 
-error: value of type `m::S` does not have the `drop` ability
-   ┌─ tests/bytecode-generator/wildcard3.move:10:13
-   │
-10 │         let _ = s;
-   │             ^ implicitly dropped here since it is no longer used
+error: local `s` of type `m::S` does not have the `drop` ability
```

Comment:
Wrong location?

Maybe we could just say "implicitly dropped since it is no longer used" (removed the word "here", which maybe only adds to confusion about where exactly the drop happens).